src/webui/client/modules/performances/entities/node.js

```diff
@@ -53,7 +53,7 @@ define(['application', 'backbone', 'lib/api', 'jquery', 'supermodel'],
                 kfanimation: {
                     label: 'KF Animation',
                     properties: ['kfanimation', 'fps', 'blender_mode'],
-                    defaultValues: {blender_mode: 'no', fps: 24}
+                    defaultValues: {blender_mode: 'no', fps: 24, duration: 1}
```

Comment:
kfanimation duration is not required as default value. Instead it should be calculated on init